### PR TITLE
Finished header component

### PR DESCRIPTION
**Pull request by michael-alade**
This PR transform the header html in the static_build to a react component.

header

### CODE REVIEW

**Review by abiodun0 — CHANGES_REQUESTED** [review]
I see no tests!.. Please write comprehensive tests for each and every of this components

File: src/components/icons/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export { default as ExpandIcon } from './expandIcon/expandIcon';
```
Comment (abiodun0) [inline src/components/icons/index.js]: Dont use this format. Use as it was written before i.e
```javascript
export ExpandIcon from './expandIcon/expandIcon'
````

File: src/components/index.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
-export Sample from './sample/sample';
-export Header from './header/header';
-
+export { default as Sample } from './sample/sample';
```
Comment (abiodun0) [inline src/components/index.js]: Same comment as above

File: src/components/sample/sample.test.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { render } from 'enzyme';
 import Sample from './sample';
 
 describe('Sample', () => {
   it('should have the content', () => {
-    const sample = shallow(<Sample />);
+    const sample = render(<Sample />);
```
Comment (abiodun0) [inline src/components/sample/sample.test.jsx]: Shallow rendering would do here.. if it's to contain you are checking for here. it's faster and doesn't render the full dom.

File: src/components/selector/selector.scss [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    height: $tvn-fixed-top-menu-height;
+    overflow: hidden;
+    padding: 0 10% !important;
+    border: 0;
+    border-top: $tvn-stroke;
+    cursor: pointer;
+
+    .header, .description {
+      margin: 0;
+      width: 100%;
```
Comment (abiodun0) [inline src/components/selector/selector.scss]: Don't use magic numbers. if you are using any literals make it avaialble inside the _variables.css with their unique variable names

**Review by abiodun0 — CHANGES_REQUESTED** [review]
Some minor changes to make :)

**Review by abiodun0 — APPROVED** [review]

**Comment by andela-rsadiq** [comment]
@andela-akolawole Could you give an overview/summary of what this PR does as a body attached to the PR itself? Also, there seems to be a lot of files change for one PR. Could you possibly break it down to small units of implementation? I would be closing this PR to allow for you to make these correction. Thanks

**Pull request closed without merging**